src/interp.rs

```diff
@@ -37,9 +36,9 @@
 
 use rustc::ty::Ty;
 
-// Stores a value represented as a vector of bytes, along with its type annotation.
-// It's important to keep track of a value's type while it is being evaluated,
-// so that we know how to interpret the bytes.
+// Stores a value represented as a vector of bytes, along with its type
+// annotation.  It's important to keep track of a value's type while it is being
```

Comment:
Double space after "."

Reply:
Ah, I think this was a rogue vim setting. `set nojoinspaces` ;)